moped-editor/src/views/projects/projectView/ProjectComponentsMapEdit.js

```diff
@@ -414,7 +414,7 @@ const ProjectComponentsMapEdit = ({
       */
       name: "",
       project_id: Number.parseInt(projectId),
-      status_id: 1,
+      is_deleted: false,
```

Comment:
Not sure if we should keep this as part of the line since `is_deleted` is false by default in the database so having this in the payload of the mutation just applies the same value.

Reply:
I ended up deciding to remove `is_deleted` from the payload here since its default value is already `false`.